2019-06-06 09:48:54,881 log_Default_Display INFO   SSFERunHandler : Log initialised: Default_Display_Log
2019-06-06 10:37:18,380 log_Default_Display ERROR  object_SSFEFormMain : !!UNHAN_ERR!! ; Uncaught exception
Traceback (most recent call last):
  File "Q:\CODE\NOG\v1_3_0_6_a1\devel\RS_SRC__NOG_v1_3_0_6_a1_devel\proj_Py2_NeOGenFE_v1_3_0_6_a1_devel\src\object_SSFEFormMain.py", line 7151, in func_QComboBox_SCENARIO_FILE_Selected
    self.func_Populate_Widgits_With_Config_Data__BATCH_SCENARIO(str_Batch_Scenario_Relative_Path_And_File__Current)
  File "Q:\CODE\NOG\v1_3_0_6_a1\devel\RS_SRC__NOG_v1_3_0_6_a1_devel\proj_Py2_NeOGenFE_v1_3_0_6_a1_devel\src\object_SSFEFormMain.py", line 19089, in func_Populate_Widgits_With_Config_Data__BATCH_SCENARIO
    ''' Read the Batch_Scenario file '''
  File "Q:\CODE\NOG\v1_3_0_6_a1\devel\RS_SRC__NOG_v1_3_0_6_a1_devel\proj_Py2_NeOGenFE_v1_3_0_6_a1_devel\src\object_SSFEFormMain.py", line 22442, in func_Get_Config_Data__BATCH_SCENARIO
    ''' Get the number of loci from the file '''
ValueError: need more than 2 values to unpack
